keras_core/backend/torch/optimizer.py

```diff
@@ -0,0 +1,13 @@
+from keras_core.optimizers.base_optimizer import BaseOptimizer
+
+
+class TorchOptimizer(BaseOptimizer):
```

Comment:
Since we have more than 1 file we might want to move this to `torch/optimizers/torch_optimizer.py` (and name the others e.g. `torch_sgd.py`)

Reply:
Done.